front/src/reducers/osrdsimulation/types.ts

```diff
@@ -52,7 +52,7 @@ interface Stop {
   track_number: number;
 }
 
-export interface RouteAspect<Time = number, Color = number> {
+export interface RouteAspect<Time = number, Color = string> {
```

Comment:
By default it’s `RouteAspect<number, number>`. if changed it will affect all the places where color needs to be a number.

if you need to use the other one  (used in `consolidatedSimulation`) it’s a `RouteAspect<Date | null, string>`

I have created type aliases so we don’t have to type those params all the time https://github.com/DGEXSolutions/osrd/pull/2799